aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -165,4 +165,42 @@ RegisterOperators::~RegisterOperators() = default;
 RegisterOperators::RegisterOperators(RegisterOperators&&) noexcept = default;
 RegisterOperators& RegisterOperators::operator=(RegisterOperators&&) noexcept = default;
 
+
+CppFunction::CppFunction(KernelFunction func, std::unique_ptr<c10::FunctionSchema> schema)
+  : func_(std::move(func))
+  , schema_(std::move(schema))
+  {}
+
+Module::Module(const char* ns)
+  : ns_(ns)
+  {}
+
+// NB: lives here to reduce link time
```

Comment:
The difference in link time should be minimal, and this probably costs you an (also minimal) performance hit.

Reply:
So, in a few versions of my diff, I managed to fuck up my linker so that it took like 10G of memory and several minutes to link `libtorch_cpu.so`. I never figured exactly what was causing it though... :/

Reply:
I'll go ahead and remove this, as I haven't really established a causal relationship here.